man/systemd.netdev.xml

```diff
@@ -1027,6 +1030,53 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[HSR] Section Options</title>
+
+    <para>The [HSR] section only applies for
+    netdevs of kind <literal>hsr</literal>, and accepts the
+    following keys:</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>Ports=</varname></term>
+        <listitem>
+          <para>Specifies the underlying interfaces. This field is mandatory and must contain exactly two
+          interface names.</para>
```

Comment:
Could you extend the format that this option takes:
- mention that this can take space separated list of interface names.
- this can be specified multiple times, hence the two cases below result the same:
```
Ports=eth1 eth2
```
```
Ports=eth1
Ports=eth2
```
- mention that all previous assignments are cleared when an empty string is specified.